content/Watsonx/WatsonxData/110.md

```diff
@@ -0,0 +1,148 @@
+
+1. Follow watsonx.ai [100: Prerequisites and FAQs](/watsonx/watsonxai/100) to login to your watsonx.ai project.
+
+2. click on the hamburger icon in the upper left to expand the left-side menu and click **View all projects**
+
+  ![](./images/110/wx.ai.png)
+
+3. Select the project you used to complete your watsonx.ai labs and click on **Assets** tab.
+
+  ![](./images/110/wx.ai.project.assets.png)
+
+4. You will connect to watsonx.data instance from watsonx.ai project. Click on **New task** button from upper right and select **Connect to a data source**
+
+  ![](./images/110/wx.ai.connect.png)
+
+5. With current release of watsonx, you will use **Presto** connector to connect to data in watsonx.data. Select **Presto** and then click on **Select** button.
+
+  ![](./images/110/wx.ai.connect.presto.png)
+
+6. Fill the connection detail:
+
+> NOTE: Since it is shared environment, use unique name for conntection, you can add your name as well in the Name field. e.g. **watsonx.data connection by your First Name Last Name**
+
+   - **Name:**                    watsonx.data connection
+   - **Description:**             Connect watsonx.data using presto connector
+   - **Hostname or IP address:**  wxddev.buildlab.cloud
+   - **Port:**                    8443
+   - **Username:**                alice
+   - **Password:**                *from slack*
+   - **Port is SSL-enabled:**     Checked
+   - **SSL certificate:**         *from slack*
+
+  ![](./images/110/wx.ai.connect.presto-detail.png)
+
+7. After filling all details correctly, you can test the connection before saving the connection. Click on **Test connection**. If you see **The test was successful**. Click **Create** to save the connection information.
+
+  ![](./images/110/wx.ai.connect.presto-test.png)
+
+8. You will see saved connection with name **watsonx.data connection** in **Assets** tab
+
+  ![](./images/110/wx.ai.connect.presto-save.png)
+
+
+## Data Visualization
+
+We have setup a connection from watsonx.ai over watsonx.data. Now next step is to bring the table from watsonx.data to watsonx.ai.
+
+1. From Assets tab of selected watsonx.ai project, click on **Import assets**. Select the connection you have created **watsonx.data connection**. You can follow the selection to select table in final stage. **watsonx.data connection -> iceberg_data -> dsteam -> classic_cars**. Select **Import** button. You will notice new asset created with table name **classic_cars**
+
+  ![](./images/110/import-asset.png)
+
+2. You will notice **db2wh** catalogs are not visible on the screen because **alice** has not been given access to **db2wh** catalog.
+
+3. Click on **classic_cars** to preview the data. You will notice 0 mpg value for some of the rows in table. My objective is to prepare the data by removing cars with 0 mpg so that refined data can be shared with business user. Lets go to data refinery by clicking on **Prepare data** button from upper right. 
```

Comment:
```suggestion
3. Click on **classic_cars** to preview the data. You will notice a 0 mpg value for some of the rows in the table. My objective is to prepare the data by removing cars with 0 mpg so that refined data can be shared with business users. Let us go to the data refinery by clicking on the **Prepare data** button from the upper right. 
```